src/components/SequenceCard/SequenceCard.js

```diff
@@ -55,97 +57,133 @@ export default function SequenceCard({
           )}
         </ToggleContainer>
       </Header>
-      {tag ? (
-        <InfoIfTag>
-          <TagAndLength>
+      <MainContent>
```

Comment:
I don't like the naming of MainContent and AdditionalContent. Maybe you could try and find a more specific name?